### DISCUSSION TITLE
两个问题

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#412] [提问] [answered]
1.为什么我创了一个联机房间 但是关掉pcl2再打开联机房间不见了。。。
2.我明明买了正版java,且创建了档案 为什么还不让登陆正版微软账号？？？

**Comment by WForst-Breeze** [comment, marked as answer]
您好，很高兴为您解答~ \(￣︶￣*\))
1. 联机模块已被关闭，请检查你的PCL2版本是否为最新版，关于此问题请见 **[PCL2 / HMCL / BakaXL 对联机功能下一步发展的联合公告](https://www.bilibili.com/read/cv19553725)**。
2. 请详细描述问题，在这里给出两种问题可能的解决方案：
 (1) 若是登录进度卡住不动的话，请检查你的网络或使用VPN。
 (2) 若是PCL2提示你没有正版账号的话，请先下载 **[官方启动器](https://www.minecraft.net/zh-hans/download)** ，在此登录后再尝试在PCL2登录。
如问题仍无法解决，请尝试使用其他启动器如 **[HMCL](https://hmcl.huangyuhui.net/download/)**、**[BakaXL](https://www.bakaxl.com/)**。

**Comment by Pigeon0v0** [comment]
第一个问题上面的人已经回答了

关于微软账号问题，有的时候是微软服务器出现了一点问题，可能需要先在官网登录一次再登录启动器
